components/post/post-content.tsx

```diff
@@ -1,44 +1,24 @@
-import { Post, PostReply } from '@/utils/models';
+import { Post } from '@/utils/models';
+import Link from 'next/link';
 import { InteractionStrip } from '../shared/interaction/interaction-strip';
-import { TimeDiff } from '../shared/time-diff';
 import { UserImage } from '../shared/user-image';
 import ZoomImage from '../shared/zoom-image';
-import { UserHandle } from '../user/user-handle';
+import { UserHeader } from '../user/user-header';
 
-export const PostContent = ({
-  post,
-  replies,
-  size,
-}: {
-  post: Post;
-  replies?: PostReply[];
-  size?: 'large';
-}) => {
+export const PostContent = ({ post, size }: { post: Post; size?: 'large' }) => {
   // TODO: refactor to use classnames npm package
   const textClasses = `mt-s text-black ${size === 'large' && 'text-[20px] tracking-normal leading-relaxed'}`;
-  // const user = await GetUserById(post?.creator?.id ?? '');
-  // const displayedName = getName(user);
-  const displayedName = post.creator.username;
 
   return (
     <div>
       <div className='absolute -ml-[82px] -mt-xs'>
-        <UserImage border={true} url={post?.creator?.avatarUrl}></UserImage>
+        <Link href={`/user/${post.creator.id}`}>
+          <UserImage border={true} url={post?.creator?.avatarUrl}></UserImage>
+        </Link>
       </div>
-      <div className='mb-xs mb-font-label-l'>{displayedName}</div>
-      <div className='flex'>
-        <div>
-          <UserHandle
-            name={post?.creator?.username || ''}
-            id={post?.creator?.id}
-          ></UserHandle>
-        </div>
-        <div className='ml-s'>
-          <TimeDiff ulid={post.id}></TimeDiff>
-        </div>
-      </div>
-      <div className={textClasses}>
-        <a className='break-words' href={`/post/${post.id}`}>{post?.text}</a>
+      <UserHeader post={post} showTime={true} useLarge={true}></UserHeader>
+      <div className={textClasses} style={{ wordBreak: 'break-word' }}>
```

Comment:
instead of using `style={{ wordBreak: 'break-word' }}` you could add the tailwindclass `break-words` to the textClasses variable

Reply:
I tried. It will work initially, but won't adjust if window size is adjusted (...when responsive design is tested in chrome dev tools).